onnx/cpp2py_export.cc

```diff
@@ -573,6 +573,21 @@ PYBIND11_MODULE(onnx_cpp2py_export, onnx_cpp2py_export) {
     return py::bytes(out);
   });
 
+  // inline_selected_functions: Inlines all functions specified in function_ids, unless
+  // exclude is true, in which case it inlines all functions except those specified in
+  // function_ids.
+  inliner.def(
+      "inline_selected_functions",
```

Comment:
The documentation should include the parameter definition. *exclude* is not intuitive.